.github/workflows/build_and_test.yml

```diff
@@ -462,61 +462,61 @@ jobs:
       run: |
         ./build/sbt -Pyarn -Pmesos -Pkubernetes -Phive -Phive-thriftserver -Phadoop-cloud -Pkinesis-asl -Phadoop-2.7 compile test:compile
 
-  tpcds-1g:
-    name: Run TPC-DS queries with SF=1
```

Comment:
Can we simply skip it for now? e.g.) `if: ${{ '1' == '0' }}`

Reply:
SGTM, thanks~